src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,36 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
+    private String repeatPassword;
```

Comment:
add size constraints

Reply:
It is not necessary because I valid passwords with annotation @Password